src/mgos_imu_icm20948.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Google Inc.
```

Comment:
Google didn't write this so it does not get the copyright.
Feel free to write your name here so it's clear this is your contribution.